slack/slack_api.py

```diff
@@ -242,6 +246,21 @@ async def fetch_user_info(self, user_id: str):
             raise SlackApiError(self.workspace, method, response, params)
         return response
 
+    async def _set_user_profile(self, profile: dict):
```

Comment:
I see there's a TypedDict class `SlackProfileField`, but it expects two keys, is the second one `alt` necessary? I created a specific class for this case, but I wonder if I shouldn't just reuse that (and make the second key optional).

Reply:
That type is for the response from the API, which isn't exactly the same as the required input, so yeah, we have to create a new type here. Since we're only using it for status updates, it would be sufficient to just create a `TypedDict` with `status_emoji` and `status_text`. Alternatively the fields listed on https://api.slack.com/methods/users.profile.set should be included.